smart-contracts/scripts/deployments/governor.js

```diff
@@ -54,14 +51,7 @@ async function main() {
   )
 
   // get UDT token address
-  let tokenAddress
-  if (networkName !== 'localhost') {
-    const [UDTInfo] =
-      OZ_SDK_EXPORT.networks[networkName].proxies[
-      'unlock-protocol/UnlockDiscountToken'
-      ]
-    tokenAddress = UDTInfo.address
-  }
+  let tokenAddress = '0xaB82D702A4e0cD165072C005dc504A21c019718F'
```

Comment:
Should we use a param for this and not hardcode it?

Reply:
yes, my bad